airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/pipeline.py

```diff
@@ -100,6 +100,7 @@ async def _run(self, built_containers_per_platform: Dict[Platform, Container]) -
         ], "This step can only run for Python connectors."
         built_container = built_containers_per_platform[LOCAL_BUILD_PLATFORM]
         pip_freeze_output = await built_container.with_exec(["pip", "freeze"], skip_entrypoint=True).stdout()
+        # TODO CDK version for manifest only connectors
```

Comment:
Why does it have to change? In any case we might reconsider the source of truth here: the [dependency list from the connector insights](https://storage.googleapis.com/prod-airbyte-cloud-connector-metadata-service/connector_insights/source-recurly/latest/insights.json) is more exhaustive as it's based on the SBOM.